src/javascripts/legacy_app.js

```diff
@@ -18,7 +18,8 @@ var App = {
     'app.created': 'init',
     'getMe.done': 'renderMain',
     'click .js-modalButton': 'showModal',
-    'modal.close': 'logClosedModal'
+    'modal.close': 'logClosedModal',
```

Comment:
Let's add a note in [Migrating to v2](https://developer.zendesk.com/apps/docs/apps-v2/migrating) mentioning the scaffold doesn't support modals for now.